<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fr_mentor</class>
 <widget class="QWidget" name="fr_mentor">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mentor Interview</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Genel √ßer√ßeve */

QWidget {
    background-color: #079af5; /* Deniz mavisi koyu ton */	
}

/* QLabel (Logo ve ba≈ülƒ±klar i√ßin) */
QLabel {
    background-color: transparent;  /* Tam ≈üeffaf */
	color: rgb(31, 113, 186);
	font-weight: bold; 
    font-size: 25px;             /* Yazƒ± b√ºy√ºk ‚Äì butonu dolduracak gibi */
}

/* QPushButton */
QPushButton {
    background-color: #079af5; /* Widget arka planƒ±ndan biraz daha a√ßƒ±k deniz mavisi */
    color: white;
}

QPushButton:hover {
    background-color: #007ab8; /* Hover i√ßin biraz daha parlak */
	color: rgb(0, 0, 255);
}

QPushButton:pressed {
    background-color: #004a68; /* Basƒ±lƒ± halde koyu ton */
}

/* QLineEdit */
QLineEdit {
    background-color: #eaf6fb; /* Ana rengin beyaza √ßok yakƒ±n tonu */
    border: 1px solid #079af5;
    border-radius: 4px;
    padding: 4px 8px;
    color: #000000;
}

/* QComboBox */
QComboBox {
    background-color: #eaf6fb; /* Textbox ile aynƒ± */ 
    color: #000000;
    border: 1px solid #079af5;
    padding: 4px 8px;
    border-radius: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #005f86;
}



/* QTableWidget veya QTableView */
QTableWidget, QTableView {
    background-color: #eaf6fb; /* veya √∂rnek: #f0f0f0 (a√ßƒ±k gri) */
    color: black;
    gridline-color: #005f86;
    alternate-background-color: #f9f9f9;	
	border: 1px solid #079af5;
    selection-background-color: #00aaff;
    selection-color: black;
	border-radius: 1px;
}

/* Tablo ba≈ülƒ±k alanlarƒ± */
QTableWidget QTableCornerButton::section,
QHeaderView::section {
    background-color: #079af5; /* Header daha a√ßƒ±k deniz mavisi */
    color: white;
    font-weight: bold; 
    padding: 2px;
}



/* Frame */
QFrame {
    background-color: #58b9f5; /* Belirgin, orta a√ßƒ±k deniz mavisi */
    border: none;
    border-radius: 10px;

}

/* Scrollbar (isteƒüe baƒülƒ±) */
QScrollBar:vertical, QScrollBar:horizontal {
    background: #003f5c;
    width: 10px;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #005f86;
    border-radius: 5px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,10,80,5">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="logo_label">
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../resorces/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="title_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">


</string>
          </property>
          <property name="text">
           <string>Mentor Meeting</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>110</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="30,50,60,30">
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="all_applicaitons_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>3</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>üóÇÔ∏è All Applications</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QComboBox" name="decision_combobox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>293</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>VIT Projesinin Tamamƒ±na Katƒ±lmasƒ± Uygun Olur</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VIT Projesini ile IT Eƒüitimi Alanƒ±na Y√∂nlendirilmesi</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="search_edit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>32</height>
           </size>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="search_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>üîç Search</string>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="applications_table">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>805</height>
         </size>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">    background-color: #eaf6fb; /* veya √∂rnek: #f0f0f0 (a√ßƒ±k gri) */
    color: black;
    gridline-color: #005f86;
    alternate-background-color: #f9f9f9;	

    selection-background-color: #00aaff;
    selection-color: black;
	border-radius: 0px;
</string>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </row>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>G√∂r√º≈üme Tarihi</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ad Soyad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mentor</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IT Bilgisi</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Yogunluk</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Yorumlar</string>
         </property>
        </column>
        <item row="4" column="0">
         <property name="text">
          <string>ascac</string>
         </property>
        </item>
        <item row="4" column="1">
         <property name="text">
          <string>asc</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="back_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>üîô Back to Preferences</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exit_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>‚ùå Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
